sdk/template/azure-template/CHANGELOG.md

```diff
@@ -1,6 +1,64 @@
 # Release History
 
-## 1.0.0-beta.8 (Unreleased)
+## 1.0.0-beta.22 (2020-01-08)
```

Comment:
?  do we want this?

Reply:
I don't think there's anything wrong with it. I should go through and clean up all the releases of `azure-template` cluttering up our repo after this merges.